README.md

```diff
@@ -39,6 +46,8 @@ We welcome contributions to enhance these tutorials.
 
 ### Propose changes
 
+> **Note:** Follow the below contribution steps if you plan to create a tutorial that is independant from the offical Grafana Labs documentation. Otherwise follow these [steps](./docs/transformer.md#generate-a-tutorial).
```

Comment:
A little rewrite to avoid directional language (accessibility) and use simple words. I've also added a little more context for the link to the transformer documentation.

```suggestion
> **Note:**
> Use the following contribution steps if you want to create a tutorial that's not also part of Grafana Labs documentation.
> Otherwise follow these [steps](./docs/transformer.md#generate-a-tutorial) to generate a tutorial from that documentation.
```